src/webui/www/private/views/preferences.html

```diff
@@ -1929,7 +1945,7 @@
             // Validate form data
             // Downloads tab
             // When adding a torrent
-            settings.set('create_subfolder_enabled', $('createsubfolder_checkbox').getProperty('checked'));
+            settings.set('torrent_content_layout', $('contentlayout_select').getSelected()[0].getProperty('value'));
```

Comment:
Can't we use `$('contentLayout_select').value` instead of `$('contentlayout_select').getSelected()[0].getProperty('value')`?

Reply:
Same here
https://github.com/qbittorrent/qBittorrent/blob/a579b4a51999d7148ac7c1f3191bee5e95b6816c/src/webui/www/private/views/preferences.html#L2120